docs/abridged_jacquard_dataset.md

```diff
@@ -0,0 +1,47 @@
+# README File
```

Comment:
I've put the documentation for the dataset in this repo too, seems like the appropriate place.